### DISCUSSION TITLE
request issue on CL api ?

### DISCUSSION THREAD

**Discussion by purnima110895** [freelawproject/courtlistener#4112] [Q&A] [answered]
Getting below error while running CL API -

{'detail': 'Request was throttled. Expected available in 21227 seconds.'}
requests.exceptions.SSLError: HTTPSConnectionPool(host='www.courtlistener.com', port=443): Max retries exceeded with url: /api/rest/v3/courts/?fields=id%2Cfull_name%2Curl&page=132 (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1002)')))

Please help me with suggestions for this error , other than proxy if any.
@mlissner

**Comment by mlissner** [comment]
That error doesn't ring any bells for me. Maybe if you provide more code or a traceback?

**Reply by purnima110895** [reply, marked as answer]
This error comes when max hits are there on API call.
I got it resolved through authorization using login credentials. 
Its described in Limitations of API's , only 5000 queries per hour is allowed to hit.

**Comment by purnima110895** [comment]
Login creds can help to get 5000 query hits her hour.
